### PR TITLE
docs: Update reference to @jitclass location

### PR DESCRIPTION
**Pull request by dnadlinger**

### CODE REVIEW

File: docs/source/user/jitclass.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Compiling Python classes with ``@jitclass``
   exposed or implemented, yet.
 
 
-Numba supports code generation for classes via the :func:`numba.jitclass`
+Numba supports code generation for classes via the :func:`numba.experimental.jitclass`
```
Comment (stuartarchibald) [inline docs/source/user/jitclass.rst:13]: Please line wrap to 80 characters, thanks!

**Review by stuartarchibald — COMMENTED** [review]
Thanks for the patch, one minor thing to resolve else looks good.

File: docs/source/user/jitclass.rst (hunk 1 above)
Comment (dnadlinger) [reply docs/source/user/jitclass.rst:13]: Happy to do so; it will just increase the diff size.

**Comment by dnadlinger** [comment]
Rewrapped (only text change is still adding `.experimental` to the reference).

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch!

File: docs/source/user/jitclass.rst (hunk 1 above)
Comment (stuartarchibald) [reply docs/source/user/jitclass.rst]: Thank you :)

**Pull request merged**